PICS.xcodeproj/xcshareddata/xcschemes/PICS.xcscheme

```diff
@@ -95,7 +95,7 @@
          </CommandLineArgument>
          <CommandLineArgument
             argument = "--useFirebaseEmulator"
-            isEnabled = "NO">
+            isEnabled = "YES">
```

Comment:
Here we want to disable the tag as we will use the actual firebase in the deployment

Reply:
Yes, you're totally right! Thank you so much - just changed it right now :)